.github/workflows/native-unix.yml

```diff
@@ -189,6 +189,165 @@ jobs:
           env BUILD_ALL=0 BUILD_DRIVER_MANAGER=1 ./ci/scripts/cpp_build.sh "$(pwd)" "$(pwd)/build"
           env BUILD_ALL=0 BUILD_DRIVER_MANAGER=1 ./ci/scripts/cpp_test.sh "$(pwd)" "$(pwd)/build"
 
+  # ------------------------------------------------------------
+  # C/C++ driver service integration tests
```

Comment:
There's another PR refactoring this...maybe I'll merge that first so we can rebase here?

Reply:
Sounds good to me. I'm still waiting for Snowflake to merge the PR on their end, so let me know when that gets merged and i'll rebase this for it.

Reply:
I just merged it!

Reply:
rebased and pushed!

Reply:
Shouldn't this all go in the integration.yml pipeline now? (I split it out a while back to prevent this pipeline from getting unwieldy.)